services/src/api/handlers/datasets.rs

```diff
@@ -251,6 +258,49 @@ pub async fn get_loading_info_handler<C: ApplicationContext>(
     Ok(web::Json(dataset))
 }
 
+/// Updates the dataset's symbology
+#[utoipa::path(
+    tag = "Datasets",
+    post,
```

Comment:
```suggestion
    put,
```

https://stackoverflow.com/questions/630453/what-is-the-difference-between-post-and-put-in-http

you update a symbology of a fixed resource and the output is always the same

Reply:
6215207e120a5308d1672fe136b500bc4e1ab300